lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
Hm... I've changed the typings to the code you suggested above and it appears like TypeScript prefers the `[{ all: true }]` array above everything else for some weird reason.

The only way I could get `tsc` to accept the two examples in `test/include.ts` was by changing the typings to:

```TypeScript
export type Includeables = Includeable[] | IncludeAll[];

export interface IncludeAll {
  all: true;
}
```
Your original suggestion of 
```TypeScript
export type Includeables = Includeable[] | [{ all: true }];
```
results in a `tsc` error:
```
test/include.ts(10,9): error TS2345: Argument of type '{ include: [{ model: typeof AssociatedModel; where: { state: Col; }; limit: number; separate: boo...' is not assignable to parameter of type 'FindOptions'.
  Types of property 'include' are incompatible.
    Type '[{ model: typeof AssociatedModel; where: { state: Col; }; limit: number; separate: boolean; order...' is not assignable to type 'Includeables'.
      Type '[{ model: typeof AssociatedModel; where: { state: Col; }; limit: number; separate: boolean; order...' is not assignable to type '[{ all: true; }]'.
        Type '{ model: typeof AssociatedModel; where: { state: Col; }; limit: number; separate: boolean; order:...' is not assignable to type '{ all: true; }'.
          Object literal may only specify known properties, and 'model' does not exist in type '{ all: true; }'.
```

Not sure how to proceed from here on to be honest... Do you want to allow potential "errors" by changing `Includeables` to `Includeable[] | IncludeAll[]`?